ecs-cli/modules/version/version.go

```diff
@@ -22,7 +22,7 @@ package version
 // repository. Only the 'Version' const should change in checked-in source code
```

Comment:
Is this comment still accurate?

Reply:
I don't believe so. We've been manually bumping the version (I think) for a while. I modeled this PR off the last one @SoManyHs merged in.